src/buckets/components/createBucketForm/UpdateBucketOverlay.tsx

```diff
@@ -131,10 +156,58 @@ const UpdateBucketOverlay: FunctionComponent<Props> = ({
     }
   }
 
+  const isValid = () => {
+    // are there measurement schemas?
+    const haveSchemas =
+      !!newMeasurementSchemaRequests && !!newMeasurementSchemaRequests.length
```

Comment:
I think it might be more explicit and clear if instead of doing a casted boolean check,  you check if it's an array:

```ts
const haveSchemas =
      Array.isArray(newMeasurementSchemaRequests) && newMeasurementSchemaRequests.length